src/behavior.scm

```diff
@@ -291,17 +291,17 @@
 			)
 			;; Were we interacting with someone else?  If so, then
 			;; maybe glance at the location of the person who left.
-			(SequentialAnd
-				(DefinedPredicate "Is interacting with someone?")
-				(SequentialOr
-					(NotLink (DefinedPredicate "dice-roll: glance lost face"))
-					(FalseLink (DefinedSchema "glance at lost face"))
-					(Evaluation (GroundedPredicate "scm: print-msg")
-						(ListLink (Node "--- Glance at lost face"))))
-				(TrueLink)
-			)
-			(Evaluation (GroundedPredicate "scm: print-msg")
-				(ListLink (Node "--- Ignoring lost face")))
+			;(SequentialAnd
+			;	(DefinedPredicate "Is interacting with someone?")
+			;	(SequentialOr
+			;		(NotLink (DefinedPredicate "dice-roll: glance lost face"))
+			;		(FalseLink (DefinedSchema "glance at lost face"))
+			;		(Evaluation (GroundedPredicate "scm: print-msg")
+			;			(ListLink (Node "--- Glance at lost face"))))
+			;	(TrueLink)
+			;)
+			;(Evaluation (GroundedPredicate "scm: print-msg")
+			;	(ListLink (Node "--- Ignoring lost face")))
 			(TrueLink)
```

Comment:
Well, yeah!  The whole point is that she is supposed to *remember things about the past* including remembering that, not very long ago, someone had been standing there, and had moved away.  You can't just wipe her brain, can't wipe all of her memory, just because someone is not visible any more.  So, yes, we need to know about the location of "lost faces", and remember that for at least 5 seconds or so.  After that, amnesia is OK, for now.